app/api/helpers/scheduled_jobs.py

```diff
@@ -140,3 +140,13 @@ def send_event_fee_notification_followup():
                                                         app_name,
                                                         link,
                                                         incomplete_invoice.event.id)
+
+
+def expire_pending_tickets_after_one_day():
+    from app import current_app as app
```

Comment:
Instead of importing `timedelta`, I believe that `(Order.create_at - datetime.today()).days>100` would be better. Otherwise, this looks fine. One question, if we're running the scheduled job every 5 hours, why do we need check for orders which were placed 100 days before?

Reply:
@mrsaicharan1 Will implement the code suggestion. and the 100 days was a mistake, correcting it